go/vt/vttablet/tabletserver/repltracker/poller.go

```diff
@@ -20,11 +20,15 @@ import (
 	"sync"
 	"time"
 
+	"vitess.io/vitess/go/stats"
+
 	"vitess.io/vitess/go/vt/mysqlctl"
 	vtrpcpb "vitess.io/vitess/go/vt/proto/vtrpc"
 	"vitess.io/vitess/go/vt/vterrors"
 )
 
+var replicationLagGauges = stats.NewGauge("replicationLagSec", "replication lag in seconds")
```

Comment:
sorry to be nitpicky, but could you rename the variable? It can be the same: `replicationLagSec` or even rename both the variable and gauge to `replicationLagSeconds`.

Reply:
Yep, good catch. I was going to do that initially as well :-)